### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by George7h**
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling
- Create a new game with the name of your choice by using the API.
- Make sure that you saved the ID of your game that will be returned by API.
- Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the AP

### CODE REVIEW

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @George7h,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉

# Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Leaderboard is set up correctly 👍🏽
✅ The app launches successfully 👍🏽
✅ App is beautifully styled and well-structured code 👍🏽

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/modules/list.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+//     user: 'Name: ',
+//     score: 77,
+//   },
+//   {
+//     user: 'Name: ',
+//     score: 42,
+//   },
+// ];
+
+// const scoreList = () => {
```
Comment (algerina) [inline src/modules/list.js:32]: - [ Optional] Kindly remove the commented code, your app will look more professional.

File: src/modules/createGame.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+// const gameName = 'Leaderboardgh7Test2';
+// createGame(gameName)
+//   .then(gameId => {
+//     if (gameId) {
+//       console.log('Game created with ID:', gameId);
+//     } else {
+//       console.log('Failed to create game.');
+//     }
+//   });
```
Comment (algerina) [inline src/modules/createGame.js:31]: - [ Optional] I recommend removing all the commented code that way your project will look amazing.

**Review by algerina — APPROVED** [review]
Hi @George7h ,

Great job so far!  

May I highlight the following: 
- [x] Your  have a descriptive PR and commit history👏  ✔️ 
- [x] Your project looks amazing✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**